avocado/utils/service.py

```diff
@@ -118,8 +118,9 @@ def method(cmd_result):
             # If service is stopped, exit_status is also not zero.
             # So, we can't use exit_status to check result.
             output = cmd_result.stdout.lower()
+            error = cmd_result.stderr.lower()
             # Returns None if XXX is unrecognized.
-            if re.search(r"unrecognized", output):
+            if re.search(r"unrecognized", error):
```

Comment:
Nice catch, I'm only wondering if this message is sent to stderr on all versions. Shouldn't we check output+error just to be sure?